syntax = "proto3";

package mavlink.msg.common;

message CommandAck {
  uint32 command = 1; /*<  Command ID (of acknowledged command)*/
  uint32 result = 2; /*<  Result of command*/
  uint32 progress = 3; /*<  Also used as result_param1, it can be set with an enum containing the errors reasons of why the command was denied, or the progress percentage when result is MAV_RESULT_IN_PROGRESS (UINT8_MAX if the progress is unknown)*/
  sint32 result_param2 = 4; /*<  Additional parameter of the result, example: which parameter of MAV_CMD_NAV_WAYPOINT caused it to be denied*/
  uint32 target_system = 5; /*<  System ID of the target recipient This is the ID of the system that sent the command for which this COMMAND_ACK is an acknowledgement.*/
  uint32 target_component = 6; /*<  Component ID of the target recipient This is the ID of the system that sent the command for which this COMMAND_ACK is an acknowledgement.*/
}

